stuff/cron/aggregate_feedback.py

```diff
@@ -441,15 +446,31 @@ def aggregate_reviewers_feedback_for_problem(
                                `Tags` AS `t`
                            WHERE
                                `t`.`name` = %s;""",
-                    (problem_id, most_voted_category))
+                    (problem_id, most_voted_level))
+
+        # Add new tags for problem
+        if len(topic_tag_votes) != 0:
+            cur.executemany("""INSERT IGNORE INTO
+                                   `Problems_Tags`(`problem_id`, `tag_id`,
+                                                   `source`)
+                               SELECT
+                                   %s AS `problem_id`,
+                                   `t`.`tag_id` AS `tag_id`,
+                                   'quality' AS `source`
+                               FROM
+                                   `Tags` AS `t`
+                               WHERE
+                                   `t`.`name` = %s;""",
+                            (list(map(lambda tag: (problem_id, tag),
+                                      topic_tag_votes))))
```

Comment:
nit: `list`, `map`, y `lambda` se pueden juntar en una sóla comprensión de lista que es más legible:

```suggestion
                            [(problem_id, tag) for tag in topic_tag_votes])
```

Reply:
Listo